udp/udp_darwin.go

```diff
@@ -6,17 +6,61 @@ package udp
 // Darwin support is primarily implemented in udp_generic, besides NewListenConfig
```

Comment:
I'm late, but I think this comment is no longer true?

Reply:
It is, nice catch. Will remove in #1427